network/server_test.go

```diff
@@ -173,6 +173,19 @@ func TestNat(t *testing.T) {
 	})
 	defer srv.Close()
 
+	t.Run("there should be multiple listening addresses", func(t *testing.T) {
+		listenAddresses := srv.host.Network().ListenAddresses()
+
+		assert.GreaterOrEqual(t, len(listenAddresses), 1)
```

Comment:
Since this test states that there should be multiple listening addresses, I guess we should expect strictly "higher than 1":
```suggestion
		assert.Greater(t, len(listenAddresses), 1)
```